### PR TITLE
 net-mgmt/telegraf: fixed #2369 if statements in wrong spot and wrong model form

### LINKED ISSUE #2369: Suricata alerts not showing up in telegraf config
**Important notices**
Before you add a new report, we ask you kindly to acknowledge the following:

[-] I have read the contributing guide lines at https://github.com/opnsense/plugins/blob/master/CONTRIBUTING.md

[-] I have searched the existing issues and I'm convinced that mine is new.

[-] The title contains the plugin to which this issue belongs

**Describe the bug**
Intrusion Detection Alerts input not showing in ```/usr/local/etc/telegraf.conf```

**To Reproduce**
1. Go to Services > Telegraf > Input
2. Check Intrusion Detection Alerts and save
3. Restart to Telegraf
4. ssh into firewall
5. cat /usr/local/etc/telegraf.conf

**Expected behavior**
Checking the box places the following into ```/usr/local/etc/telegraf.conf ```
```
[[inputs.tail]]
  data_format = "json"
  files = ["/var/log/suricata/eve.json"]
  name_override = "suricata"
  tag_keys = ["event_type","src_ip","src_port","dest_ip","dest_port"]
  json_string_fields = ["*"]

```

**Environment**
OPNsense 21.1.5 (amd64, LibreSSL).
Proxmox VM and libvirt virtual machines
Network Intel

### PR DESCRIPTION
**Pull request by tiny6996**
The if statements were out of order in the telegraf.conf file and this feature was using the output the model instead of the input model

### CODE REVIEW

File: net-mgmt/telegraf/src/opnsense/service/templates/OPNsense/Telegraf/telegraf.conf [hunk 1]
```diff
@@ -247,11 +247,13 @@
 
 {% if helpers.exists('OPNsense.telegraf.input.ntpq') and OPNsense.telegraf.input.ntpq == '1' %}
 [[inputs.ntpq]]
-{%   if helpers.exists('OPNsense.telegraf.input.ntpq_dns_lookup') and OPNsense.telegraf.input.ntpq_dns_lookup == '1' %}
+{% if helpers.exists('OPNsense.telegraf.input.ntpq_dns_lookup') and OPNsense.telegraf.input.ntpq_dns_lookup == '1' %}
```
Comment (mimugmail) [inline net-mgmt/telegraf/src/opnsense/service/templates/OPNsense/Telegraf/telegraf.conf:250]: Please reimplement the indent here

Comment (fichtner) [inline net-mgmt/telegraf/src/opnsense/service/templates/OPNsense/Telegraf/telegraf.conf:250]:
```suggestion
{%   if helpers.exists('OPNsense.telegraf.input.ntpq_dns_lookup') and OPNsense.telegraf.input.ntpq_dns_lookup == '1' %}
```

File: net-mgmt/telegraf/pkg-descr [hunk 2]
```diff
@@ -11,6 +11,9 @@ WWW: https://www.influxdata.com/time-series-platform/telegraf/
 
 Plugin Changelog
 ================
+1.10.1
```
Comment (fichtner) [inline net-mgmt/telegraf/pkg-descr:14]:
```suggestion

1.10.1
```

File: net-mgmt/telegraf/pkg-descr [hunk 3]
```diff
@@ -11,6 +11,9 @@ WWW: https://www.influxdata.com/time-series-platform/telegraf/
...
+
+* Fix Suricata input controller being the output section and incorrect in statement in conf
```
Comment (fichtner) [inline net-mgmt/telegraf/pkg-descr:17]:
```suggestion
* Fix Suricata input controller being the output section and incorrect statement
```

**Comment by fichtner** [comment]
@mimugmail ready to go?

**Comment by mimugmail** [comment]
Do it :)

**Comment by fichtner** [comment]
Merged, thanks!

**Pull request merged**